controllers/enable_mesh.go

```diff
@@ -112,10 +112,15 @@ func serviceMeshIsNotEnabled(meta metav1.ObjectMeta) bool {
 }
 
 func (r *OpenshiftServiceMeshReconciler) findIstioIngress(ctx context.Context) (routev1.RouteList, error) {
+	meshNamespace := "istio-system"
+	if env, defined := os.LookupEnv("MESH_NAMESPACE"); defined {
+		meshNamespace = env
+	}
+
```

Comment:
Two hints here:

- it's already 2nd place we call this lookup - maybe worth extracting it to its own function?
- similarly, env variables names can be moved to consts (and re-used in tests as well)

Reply:
Can consts be dynamic like this? Or would they have to be vars?

Reply:
I see, thanks for the clarification :+1: